test/multiverse/suites/concurrent_ruby/concurrent_ruby_instrumentation_test.rb

```diff
@@ -0,0 +1,76 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
```

Comment:
I know there's cleanup left to do on this file, however, I think we could also do some refactoring to DRY things up.